docs/data-integrations/apache-ignite.mdx

```diff
@@ -0,0 +1,50 @@
+---
+title: Apache Ignite
+sidebarTitle: Apache Ignite
+---
+
+This is the implementation of the Apache Ignite data handler for MindsDB.
+
+[Apache Ignite](https://ignite.apache.org/docs/latest/) is a distributed database for high-performance computing with in-memory speed.
+
+## Implementation
+
+This handler is implemented using the `pyignite` library, the Apache Ignite thin (binary protocol) client for Python.
+
+The required arguments to establish a connection are as follows:
+
+* `host` is the host name or IP address of the Apache Ignite cluster's node.
+* `port` is the TCP/IP port of the Apache Ignite cluster's node. Must be an integer.
```

Comment:
Please list all parameters here, including host, port, username, password, and schema.

Reply:
Apologies for missing this @martyna-mindsdb. It's done now.

Reply:
Thank you @MinuraPunchihewa